x/migration/keeper/msg_server_import_morse_claimable_accounts.go

```diff
@@ -0,0 +1,57 @@
+package keeper
+
+import (
+	"context"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	migrationtypes "github.com/pokt-network/poktroll/x/migration/types"
+)
+
+func (k msgServer) ImportMorseClaimableAccounts(ctx context.Context, msg *migrationtypes.MsgImportMorseClaimableAccounts) (*migrationtypes.MsgImportMorseClaimableAccountsResponse, error) {
+	sdkCtx := sdk.UnwrapSDKContext(ctx)
+	logger := sdkCtx.Logger().With("method", "CreateMorseAccountState")
+
+	if msg.GetAuthority() != k.GetAuthority() {
+		err := migrationtypes.ErrUnauthorized.Wrapf("invalid authority address (%s)", msg.GetAuthority())
+		return nil, status.Error(codes.PermissionDenied, err.Error())
+	}
+
+	// Validate the import message.
+	if err := msg.ValidateBasic(); err != nil {
+		logger.Info(err.Error())
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	// Check if MorseClaimableAccounts have already been imported.
+	if morseClaimableAccounts := k.GetAllMorseClaimableAccounts(sdkCtx); len(morseClaimableAccounts) > 0 {
```

Comment:
Is there ` TODO_MAINNET` opportunity to just retrieve one instead of reading all of them from disk?